### PR TITLE
Add Service Now Stream Connector

### PR DESCRIPTION
**Pull request by Moujimouja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,3 +131,55 @@ An HTTP events collector has be configured in data entries.
 
 ![alt text](pictures/splunk.png "Splunk configuration")
 
+# Service Now
+
+he stream connector sends the check results received from Centreon Engine to ServiceNow. Only the host and service check results are sent.
```
Comment (Pontissalien) [inline README.md]: Manque un T en début de phrase je suppose

Comment (Moujimouja) [reply README.md]: Thanks

**Comment by Pontissalien** [comment]
I think there is a typo on this file name : connector-servicenow-requet.lua (the "s" of request is missing)

File: servicenow/connector-servicenow-requet.lua [hunk 2]
```diff
@@ -0,0 +1,158 @@
+local http = require("socket.http")
+local ltn12 = require("ltn12")
+local JSON = (loadfile "JSON.lua")()
+
```
Comment (bouda1) [inline servicenow/connector-servicenow-requet.lua]: This library looks not very useful... Broker api provides two methods json_encode and json_decode.
Is this import really needed?

File: servicenow/connector-servicenow-requet.lua [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  for i,v in pairs(resp_headers) do
+    print("Header " .. i .. " : " .. v)
+  end
+  print("")
+
+  if code >= 300 then
+    error("Error when call the ServiceNow webservice")
+  end
+
+  return JSON:decode(res) -- JSON decode
```
Comment (bouda1) [inline servicenow/connector-servicenow-requet.lua]: I think we should use `return broker.json_decode(res)`

File: servicenow/connector-servicenow-requet.lua (hunk 2 above)
Comment (Moujimouja) [reply servicenow/connector-servicenow-requet.lua]: i'm not going to modify anything, I was just the messenger :) So feel free to take some times to update it, push force & integrate in master

**Comment by bouda1** [comment]
Many things have been done so far. Could one of you take a look at it ?
Thanks.

**Pull request merged**